package.json

```diff
@@ -9,6 +9,11 @@
   ],
   "author": "The Trade Desk",
   "license": "Apache 2.0",
+  "wallaby": {
+    "delays": {
+      "run": 1000
```

Comment:
I found Wallaby re-running all the tests the moment I stopped typing to be a bit laggy - increasing the delay.